package.json

```diff
@@ -27,7 +27,8 @@
     "lint:styles": "stylelint '**/*.{css,scss}' --report-needless-disables --report-invalid-scope-disables",
     "sync": "carbon-cli sync",
     "test": "cross-env BABEL_ENV=test jest",
-    "test:e2e": "cross-env BABEL_ENV=test jest --testPathPattern=e2e --testPathIgnorePatterns='examples,/packages/components/,/packages/react/'"
+    "test:e2e:jest": "cross-env BABEL_ENV=test jest --testPathPattern='e2e/' --testPathIgnorePatterns='examples,/packages/components/,/packages/react/'",
+    "test:e2e:cypress": "lerna run test:e2e:cypress"
```

Comment:
How would you feel about using the command:

```bash
yarn workspace carbon-components-react test:e2e:cypress
```

Instead of adding a new npm task? Just was curious, sometimes these get crazy with the multiple colon joined tasks that I was curious if there was a way to simplify or not.

Reply:
@joshblack I'm not opposed to it. My thought was that we may want to leave the door open to cypress testing in other packages, which the lerna command would automatically pick up any other packages that have the `test:e2e:cypress` command.